src/cli/argument-parser.ts

```diff
@@ -105,106 +132,109 @@ export default class CLIArgumentParser {
             .option('--no-color', 'disable colors in command line');
     }
 
-    _parseList (val) {
+    private _parseList (val: string): string[] {
         return val.split(',');
     }
 
-    _filterAndCountRemotes (browser) {
+    private _checkAndCountRemotes (browser: string): boolean {
         const remoteMatch = browser.match(REMOTE_ALIAS_RE);
 
         if (remoteMatch) {
             this.remoteCount += parseInt(remoteMatch[1], 10) || 1;
+
             return false;
         }
 
         return true;
     }
 
-    async _parseFilteringOptions () {
+    public async _parseFilteringOptions (): Promise<void> {
         if (this.opts.testGrep)
-            this.opts.testGrep = getGrepOptions('--test-grep', this.opts.testGrep);
```

Comment:
if ('testGrep' in this.opts)

https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-the-in-operator
```diff
- this.opts.testGrep = getGrepOptions('--test-grep', this.opts.testGrep as string);
+ this.opts.testGrep = getGrepOptions('--test-grep', this.opts.testGrep);
```